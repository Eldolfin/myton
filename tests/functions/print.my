def foo():
  pass
print(foo)

a=clock
print(a)
