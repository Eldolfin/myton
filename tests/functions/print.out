<function foo>
<built-in function clock>
